util/envDynawoAlgorithms.sh

```diff
@@ -332,17 +348,42 @@ display_environmentVariables() {
   set +x
 }
 
+config_dynawo_algorithms_3rdParties() {
+  if [ ! -d "$DYNAWO_ALGORITHMS_THIRD_PARTY_BUILD_DIR" ]; then
+    mkdir -p $DYNAWO_ALGORITHMS_THIRD_PARTY_BUILD_DIR
+  fi
+  cd $DYNAWO_ALGORITHMS_THIRD_PARTY_BUILD_DIR
+  cmake -DCMAKE_C_COMPILER:PATH=$DYNAWO_C_COMPILER \
+    -DCMAKE_CXX_COMPILER:PATH=$DYNAWO_CXX_COMPILER \
+    -DCMAKE_BUILD_TYPE:STRING=$DYNAWO_BUILD_TYPE \
+    -DDYNAWO_ALGORITHMS_HOME:PATH=$DYNAWO_ALGORITHMS_HOME \
```

Comment:
useless

Reply:
Cmake is using the cache variable, not the environment variable

Reply:
No need to be consistent if we don't need to the know the HOME to install this 3rdparty

Reply:
DYNAWO_ALGORITHMS_HOMEn is not only for 3rd parties

Reply:
yes but this cmake command is only for building 3rd parties so neither variable is used